package.json

```diff
@@ -48,7 +48,7 @@
     "@tryghost/errors": "^1.2.18",
     "bluebird": "^3.7.0",
     "cloudinary": "^1.30.0",
-    "ghost-storage-base": "^1.0.0",
+    "ghost-storage-base": "^1.1.1",
```

Comment:
There's a lot of changes besides the method rename, maybe removing the caret to lock this down to 1.0.0 would make a new release happen faster with this package.